docker/entrypoint.sh

```
@@ -14,7 +14,7 @@ else
      echo -e "Generated app key: $APP_KEY"
      echo -e "APP_KEY=$APP_KEY" > /pelican-data/.env
   else
-    echo -e "APP_KEY exists in environment, using that."
+    echo -e "APP_KEY exists in environment, using that."I c
```

Shellcheck (0.11.0)

[error] In dash, echo flags besides -n not supported.

(SC3036)